docs/image-cdn.md

```diff
@@ -1,51 +1,68 @@
+Gatsby Image CDN is a feature available since Gatsby v4.10.0. Instead of
+downloading and processing images at build time, it defers processing until
+request time. This can greatly improve build times for sites with remote images,
+such as those that use a CMS. Netlify includes full support for Image CDN, on
+all plans.
 
 ## Enabling the Image CDN
 
-To enable the Image CDN during the beta period, you should set the environment
-variable `GATSBY_CLOUD_IMAGE_CDN` to `true`.
+To enable the Image CDN, you should set the environment variable
+`NETLIFY_IMAGE_CDN` to `true`. You will also need to declare allowed image URL
+patterns in `netlify.toml`:
 
-Image CDN currently requires the beta version of Gatsby. This can be installed
-using the `next` tag:
-
-```shell
-npm install gatsby@next gatsby-plugin-image@next gatsby-plugin-sharp@next gatsby-transformer-sharp@next
+```toml
+[images]
+remote_images = [
+  'https://example1.com/*',
+  'https://example2.com/*'
+]
 ```
 
-Currently Image CDN supports Contentful and WordPress, and these source plugins
-should also be installed using the `next` tag:
+Exact URL patterns to use will depend on CMS you use and possibly your
+configuration of it.
 
-```shell
-npm install gatsby-source-wordpress@next
-```
+- `gatsby-source-contentful`:
 
-or
+  ```toml
+  [images]
+  remote_images = [
+    # <your-contentful-space-id> will be specified via `spaceId`
+    # gatsby-source-contentful option in gatsby-config
+    "https://images.ctfassets.net/<your-contentful-space-id>/*"
+  ]
+  ```
 
-```shell
-npm install gatsby-source-contentful@next
-```
+- `gatsby-source-drupal`:
+
+  ```toml
+  [images]
+  remote_images = [
+    # <your-drupal-base-url> will be speciafied via `baseUrl`
+    # gatsby-source-drupal plugin option in gatsby-config
+    "<your-drupal-base-url>/*"
+  ]
+  ```
+
+- `gatsby-source-wordpress`:
+
+  ```toml
+  [images]
+  remote_images = [
+    # <your-wordpress-url> will be specified via `url`
+    # gatsby-source-wordpress plugin option in gatsby-config
+    # with "/graphql" endpoint part being omitted
+    "<your-wordpress-url>/wp-content/uploads/*"
+  ]
+  ```
 
-Gatsby will be adding support to more source plugins during the beta period.
-These should work automatically as soon as they are added.
+Above examples are the most likely ones to be needed. However if you configure
+your CMS to host assets on different domain or path, you might need to adjust
+the patterns accordingly.
 
-## Using the Image CDN
+## How it works
 
-Your GraphQL queries will need updating to use the image CDN. The details vary
-depending on the source plugin. For more details see
-[the Gatsby docs](https://support.gatsbyjs.com/hc/en-us/articles/4522338898579)
+When using the Image CDN, Gatsby generates URLs of the form
+`/_gatsby/image/...`. On Netlify, these are served by a function that translates
+Gatsby Image CDN URLs into Netlify Image CDN compatible URL.
```

Comment:
@pieh Should we give an example of that i.e. `/.netlify/images/...`

Reply:
Yeah, we can do that. If there are any public docs about Netlify Image CDN already we could add a link too